pkg/registry/build_registry/build_registry.go

```diff
@@ -38,12 +38,12 @@ func initIANARegistry() {
 		klog.Errorf("main: %v", error)
 	}
 
-	registryFileName := "registry_IANA.go"
+	registryFileName := "./pkg/registry/registry_IANA.go"
```

Comment:
interesting.. how is the working directory root of repo and not pkg/registry/build_registry

Reply:
I tried that if we run the `go run` command in the root of repo, the working directory will change from where build_registry.go is to the root of repo. It is because of the way that os package get working directory.

Reply:
Oh ok. So we are restricting the user to run from root. Seems to be hard restriction. Maybe we could use filepath to get to the root of repo by removing pkg/registry/build_registry and use that, so it will work if we run the file from any place? https://stackoverflow.com/questions/31873396/is-it-possible-to-get-the-current-root-of-package-structure-as-a-string-in-golan

Reply:
Sure. That will be better. Thanks!